infrastructure/database/templates/03_setup_tables.template.sql

```diff
@@ -234,5 +234,12 @@ CREATE TABLE private.sensor_measurements (
     PRIMARY KEY(tree_id, type_id, timestamp)
 );
 
+CREATE TABLE private.watering_gdk (
+    tree_id TEXT REFERENCES public.trees(id),
+    amount REAL,
```

Comment:
wir haben jetzt häufig im Variablen namen die Einheit um da vorzubeugen. "rainfall_mm", "temp_c", "wind_ms".
Fänd ich hier auch gut. Sind das Liter?

Reply:
Yes its liters

Reply:
okay, added it with [71fc023](https://github.com/technologiestiftung/qtrees-ai-data/pull/24/commits/71fc0231441e86e94f3b58ca847adb283aebb5b9)